core/src/main/java/org/mule/runtime/core/internal/event/DefaultEventBuilder.java

```diff
@@ -67,6 +65,7 @@
 public class DefaultEventBuilder implements InternalEvent.Builder {
 
   private static final Logger logger = LoggerFactory.getLogger(DefaultMessageBuilder.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "logger" private field. ([squid:S1068](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7752&issues=AWm_hBy7otIXnmsUvMwW&open=AWm_hBy7otIXnmsUvMwW)

[ISSUE_KEY]: AWm_hBy7otIXnmsUvMwW